pwndbg/commands/attachp.py

```diff
@@ -71,7 +72,32 @@ def attachp(target) -> None:
                 return
 
             if len(pids) > 1:
-                print(message.warn(f"Found pids: {', '.join(pids)} (use `attach <pid>`)"))
+                pidsData.append(
+                    "{:<8} {:<15} {:<51} {:<51}".format("PID", "USER", "COMMAND", "PROCESS TREE")
```

Comment:
should use f-string literal formatting

Reply:
btw the `"{:<8} {:<15} {:<51} {:<51}"` string should probably be a constant or sth, since it is used below too.

Also we use a `tabular` module in pwndbg - maybe we should use it for rendering here?

Reply:
yep - thx for advice i did not know :)